src/components/LoginModal/LoginModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import {
+  LoginWrap,
+  LoginBox,
+  CloseBtn,
+  Content,
+  LoginBtn,
+} from './LoginModal.styles'
+
+interface ModalProps {
+  setModalOpen: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+const LoginModal = ({ setModalOpen }: ModalProps) => {
+  return (
+    <LoginWrap>
+      <LoginBox>
+        <CloseBtn onClick={() => setModalOpen(false)}>X</CloseBtn>
+        <Content>
+          <div className="logo">Logo</div>
+          <LoginBtn>Google 로그인</LoginBtn>

```

Comment:
(p4) LoginBtn 의 children 으로 문구가 넘어가는게 어색하네요. 로그인 타입을 Props 으로 넘기고, 컴포넌트 내부에서 문구를 판단하는게 관심사분리에 좀 더 적합할 것 같습니다
```suggestion
          <LoginBtn loginType={'google'} />
```